.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+## Upgrade Notes
+Does this PR prevent a zero down-time upgrade? (Assume upgrade order: Controller, Broker, Server, Minion)
+* [ ] Yes (Please label as **<code>backward-incompat</code>**)
+
+Does this PR fix a zero-downtime upgrade introduced earlier?
+* [ ] Yes (Please label this as **<code>backward-incompat</code>**)
```

Comment:
This bullet is not very intuitive to me. Are we saying that if you broke zero downtime upgrade earlier and now fixing it as part of this PR then label it backward incompat? That should be backward compatible. We recently had this scenario.

Reply:
The idea is that this flagged somehow for people to take a look at. A release may have happened between the two commits, in which case some action is needed (that happened a couple of releases ago). Or, at the minimum, the other PR needs to be flagged as backward incompat.

I am open to suggestions, but the goal is for us to know the commits during which zero downtime upgrades are broken, or if this is broken intentionally (should not happen, so needs review).

Reply:
@siddharthteotia can you suggest better wording? thanks